firecracker/packer/build-rootfs.pkr.hcl

```diff
@@ -16,7 +16,20 @@ variable "instance_type" {
 variable "region" {
   description = "In which region to build the rootfs. Must match the Base AMI's region."
   type        = string
-  default     = "us-east-1"
+  default     = "us-east-2"
```

Comment:
Why the region switch?

Reply:
As a general rule, the less hardcoded us-east-1s the better